### PR TITLE
feat(topnav): design feedback for top navigation component

### PR DESCRIPTION
**Pull request by 100stacks**
## Description

Design Review for Top Navigation

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2091

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/components/topnav/test.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <div id="vue-topNavDemo">
```
Comment (100stacks) [inline docs/components/topnav/test.html:7]: @danielmdesigns, please see this page for a full-page demo of `Top Navigation`:

https://deploy-preview-784--helix-ui.netlify.app/components/topnav/test.html

**Review by 100stacks — COMMENTED** [review]
@danielmdesigns, please add your feedback by selecting the `Files changed` tab and adding comments:

## 1. Select `Files changed` tab

Screen Shot 2020-11-05 at 1 36 02 PM

## 2. Add your comments/feedback

Screen Shot 2020-11-05 at 1 30 18 PM

## 3. Select *Review Type* and Select `Submit Review`

Screen Shot 2020-11-05 at 1 28 31 PM

File: docs/components/topnav/test.html (hunk 1 above)
Comment (danielmdesigns) [reply docs/components/topnav/test.html:7]: @100stacks Here are a few things that I noticed

- The logo area does not have a hover state
- nav-items need ~3px-4px of padding added to each side (left & right)
- I can't make a selection from the product selector, so not sure if the "active" state is available/correct
- The label "Account Number" from the dropdown should not be a hoverable/clickable link

Comment (100stacks) [reply docs/components/topnav/test.html:7]: @danielmdesigns, thanks for the feedback.  Please see below:

1. Please verify hover state for the logo area.  It appears to work in the PR preview.
1. Please add a screenshot of which nav-items need the padding.
1. This is a demo example, so we don't have it "wired up" to any logic.
1. Ok, good catch.  We'll fix this.

Comment (danielmdesigns) [reply docs/components/topnav/test.html:7]: nav-padding

Thanks for the clarification... Take a look at the attachment of the nav-item. They all appear to be ~9px of left/right padding and it should be 12px. Let me know if I am missing something but it's a little tight side-to-side

**Review by 100stacks — COMMENTED** [review]
### @danielmdesigns, please see the updates for the Top Navigation component.

#### https://deploy-preview-784--helix-ui.netlify.app/components/topnav/test.html

**Review by lalithkarikelli — APPROVED** [review]
LGTM

**Comment by danielmdesigns** [comment]
@100stacks i'm going to go ahead and approve these knowing that we have some updated changes coming down the pipe. let's go ahead and get these merged and we'll finalize once we have fabric updates available

**Review by danielmdesigns — APPROVED** [review]
LGTM! Will provide more feedback w/ updates once new Fabric changes are available

**Pull request merged**